packages/database/src/database/tokenBoundAccounts.ts

```
@@ -0,0 +1,49 @@
+  if (query?.page || query?.size) {
+    let page = parseInt(query?.page || '0')
+    let size = parseInt(query?.size || '100')
+    if (page < 0) { page = 0 }
+    if (size < 0) { size = 100 }
+    if (size > 200) { size = 200 }
+    let start = page * size
+    filter.skip = start
+    filter.take = size
+    //filter.orderBy = {}
+  }
+  let data = await prismaClient.tokenBoundAccount.findMany(filter)
```

Biome

[error] This let declares a variable that is only assigned once.

'data' is never reassigned.

Safe fix: Use const instead.


(lint/style/useConst)